test/unit/spec/signaling/v2/twilioconnectiontransport.js

```diff
@@ -104,6 +104,10 @@ describe('TwilioConnectionTransport', () => {
         assert.equal('connecting', test.transport.state);
       });
 
+      it('should set the reconnectAttempts to zero initially', () => {
```

Comment:
Not necessary to test private values. Ideally we should test the behavior.

Reply:
@manjeshbhargav - I think in for unit tests it does help with checking private values. and they are cheap.